[2021-08-08 17:20:50,142] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:20:38.409431+00:00 [queued]>
[2021-08-08 17:20:50,144] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:20:38.409431+00:00 [queued]>
[2021-08-08 17:20:50,144] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 17:20:50,144] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-08 17:20:50,145] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 17:20:50,149] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): verifica_arquivo> on 2021-08-08T20:20:38.409431+00:00
[2021-08-08 17:20:50,150] {standard_task_runner.py:52} INFO - Started process 26805 to run task
[2021-08-08 17:20:50,152] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'popula_bd', 'verifica_arquivo', '2021-08-08T20:20:38.409431+00:00', '--job-id', '268', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/popula.py', '--cfg-path', '/tmp/tmpkgmitqgo', '--error-file', '/tmp/tmpaxcos74p']
[2021-08-08 17:20:50,152] {standard_task_runner.py:77} INFO - Job 268: Subtask verifica_arquivo
[2021-08-08 17:20:50,163] {logging_mixin.py:104} INFO - Running <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:20:38.409431+00:00 [running]> on host DESKTOP-BEFA8CP.localdomain
[2021-08-08 17:20:50,177] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=vh15fleury@hotmail.com
AIRFLOW_CTX_DAG_OWNER=victor hugo
AIRFLOW_CTX_DAG_ID=popula_bd
AIRFLOW_CTX_TASK_ID=verifica_arquivo
AIRFLOW_CTX_EXECUTION_DATE=2021-08-08T20:20:38.409431+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-08T20:20:38.409431+00:00
[2021-08-08 17:20:50,177] {python.py:151} INFO - Done. Returned value was: None
[2021-08-08 17:20:50,179] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=popula_bd, task_id=verifica_arquivo, execution_date=20210808T202038, start_date=20210808T202050, end_date=20210808T202050
[2021-08-08 17:20:50,188] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-08 17:20:50,203] {local_task_job.py:149} INFO - Task exited with return code 0
